webproject/src/main/java/com/google/sps/Comparators/RoomsListingComparator.java

```diff
@@ -0,0 +1,26 @@
+package com.google.sps.Comparators;
+
+import java.util.Comparator;
+
+import com.google.sps.data.RoomProto.Room;
+
+/**
+ * A comparator to sort rooms based on how full they are.
+ * The "full" value is calculated as follows:
+ * 1. The difference of minPrice and ordersValue gives the order amount needed to hit the minPrice.
+ * 2. The delivery fee is added to the difference because it is independent of the minPrice and ordersValue.
+ * 3. The value calculated in the previous step is divided by the no of users in a room to obtain the average per
```

Comment:
do you think that average per person order value should be computed by:
getOrdersValue() / getUsersCount() ?

Reply:
please resolve if you think that this is better

Reply:
okay!